configs/permissions/vulnerability.json

```diff
@@ -1,3 +1,9 @@
 {
-    "*": ["*"]
+    "*": ["*", "read", "write"],
+    "vulnerability_results": ["read", "write"],
+    "cve.business_risk": ["write"],
+    "cve.status": ["write"],
+    "report": ["read", "generate"],
```

Comment:
having a "write" operation defined here should include ability to generate a report, correct? 
we want to standardize on verbs being used as much as possible.

Reply:
Oh I got it, fixed :)